### ISSUE TITLE
It ignore the alignment property

### ISSUE THREAD

**Issue by jasjastone** [sarbagyastha/flutter_rating_bar#112]
Hi, Thanks for making an awesome package. But

I got an issue with the rating they don't really accepts the full width and the i don't know what wrap was supported to do rather than change the rating direction to column when there is no enough space,
So here is the deal.
I got my rating ***** but i want them to occupy the full width of the screen or at least the full width of the parent container.
What have i tried,
First i thought i set the wrap to space-between or space-evenly will force them to copy full width but no it doesn't or at least i know what i doesn't since didn't get any effect on changing it(I want to mentioned that i look at source code and found that wrap is hard coded to wrap.start even change it doesn't make any difference).
Second, i try to wrap it with a Container and SizedBox and give them a full width with media query but still it doesn't occupy the full width

Long story short here is the image that i want the rating to look like:
![rating](https://github.com/sarbagyastha/flutter_rating_bar/assets/73587145/fca87686-23e1-4120-89ba-6a8fffcb6fe3)
The above is  the image that show rating that is on a full screen on an andriod app with 300x600px
If you have any idea please share.

**Comment by jasjastone** [comment]
Hi,
I manage to fix the issue as it was caused by simple passing of the `wrapAlignment` parameter to the `Material` and that's it.
Sadly i make the changes on my own if someone has face the same isse just go to the
```dart
return Material(
      color: Colors.transparent,
      child: Wrap(
        alignment: WrapAlignment.start, // the issue laid here so change it to widget.wrapAlignment
        textDirection: textDirection,
        direction: widget.direction,
        children: List.generate(
          widget.itemCount,
          (index) => _buildRating(context, index),
        ),
      ),
    );
```
Or if you want me to make a pr for i would but, from other PRs as taking so much time to review them therefore i don't think the maintainer is active or possibly abandoned the package or something.
Thank you anyways.

EDIT: The main issue was not the width rather the alignment, so you will continue to use the alignment parameter but now it will work instead of being ignored
